tests/testing_utils.py

```diff
@@ -8,6 +8,7 @@
 RUNNING_ON_CI = os.environ.get("APPVEYOR", False)
 RUNNING_TEST_FOR_MASTER_ON_CI = os.environ.get("APPVEYOR_REPO_BRANCH", "master") != "master"
 CI_OVERRIDE = os.environ.get("APPVEYOR_CI_OVERRIDE", False)
+RUN_BY_CANARY = os.environ.get("BY_CANARY", None) is not None
```

Comment:
Can we follow similar convention here? We say False on the rest.

Reply:
Updated.